### PR TITLE
updating to use API endpoints

### PR DESCRIPTION
**Pull request by lzimm**

### CODE REVIEW

File: plugins/cloud/Cloud.html [hunk 1]
```diff
@@ -86,258 +86,301 @@
...
 
-        $(document).ready(function () {
-            let config = getPluginConfig('Cloud')
-            Object.keys(config.redirectors ? config.redirectors : {}).forEach(host => {
-                REDIRECTORS[host] = {
-                    ...config.redirectors[host],
-                    destroy: destructor(config, host, config.redirectors[host])
-                }
+            $(document).ready(function () {
+                let config = getPluginConfig('Cloud')
```
Comment (privateducky) [inline plugins/cloud/Cloud.html:135]: convert any of these getPluginConfig() funcs

File: plugins/cloud/Cloud.html [hunk 2]
```diff
@@ -86,258 +86,301 @@
...
+                Object.keys(config.redirectors ? config.redirectors : {}).forEach(host => {
+                    REDIRECTORS[host] = {
```
Comment (privateducky) [inline plugins/cloud/Cloud.html:137]: Couple calls to the global REDIRECTORS still in here.

We have an API endpoint to get an internal config file ```handler.get('/internal/config/:name'``` but we don't have one to set properties in one, such as REDIRECTORS or PUBLISHERS, so we should add that.

**Review by privateducky — CHANGES_REQUESTED** [review]
2 missing pieces

File: plugins/cloud/Cloud.html (hunk 2 above)
Comment (lzimm) [reply plugins/cloud/Cloud.html]: done

File: plugins/cloud/Cloud.html (hunk 1 above)
Comment (lzimm) [reply plugins/cloud/Cloud.html]: done

**Comment by privateducky** [comment]
looks like your today one > this one

**Pull request closed without merging**